tests/example_specs/evolution_examples/7/output_manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "metadata": {
+    "base_path": "gs://bucket",
+    "run_id": "12345",
+    "component_id": "67890"
+  },
+  "index": {
+    "location": "/index"
+  },
+  "subsets": {
+    "images": {
+      "location": "/images",
```

Comment:
There is still some logic missing to update the locations of the index and subsets when they need to be rewritten. But I suggest we tackle this separately as part of https://trello.com/c/ksYMGyOw.